Sources/KeyboardShortcuts/Recorder.swift

```diff
@@ -128,6 +128,29 @@ extension KeyboardShortcuts.Recorder<Text> {
 	}
 }
 
+@available(macOS 10.15, *)
+extension KeyboardShortcuts.Recorder<Text> {
+  /**
+  - Parameter title: The title of the keyboard shortcut recorder, describing its purpose.
+  - Parameter name: Strongly-typed keyboard shortcut name.
+  - Parameter onChange: Callback which will be called when the keyboard shortcut is changed/removed by the user. This can be useful when you need more control. For example, when migrating from a different keyboard shortcut solution and you need to store the keyboard shortcut somewhere yourself instead of relying on the built-in storage. However, it's strongly recommended to just rely on the built-in storage when possible.
+  */
+  @_disfavoredOverload
+  public init<S>(
+    _ title: S,
```

Comment:
This was String before, but I changed it to match the SwiftUI.Text initializer, which takes a StringProtocol.

Reply:
I prefer to keep it a `String`. It doesn't really make much sense to pass in a substring here and I generally try to avoid StringProtocol as it has dubvious benefit in most cases.

Reply:
Fair enough. I only changed it because that is what Text takes. Easy enough to change back.